setup_hooks.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+rustup component add rustfmt-preview
+
+rustfmt_path=`find $HOME/.rustup/toolchains/ -wholename '*bin/rustfmt*'`
```

Comment:
`which rustfmt` should suffice?

Reply:
On my machine I have following:
```
SOLO@DESKTOP-0LLRSJV MINGW64 ~/Documents/Github/cv-rs (feature/FLANN)
$ find $HOME/.rustup/toolchains/ -wholename '*rustfmt*'
/c/Users/SOLO/.rustup/toolchains/stable-x86_64-pc-windows-gnu/bin/rustfmt.exe
/c/Users/SOLO/.rustup/toolchains/stable-x86_64-pc-windows-gnu/lib/rustlib/manife                         st-rustfmt-preview-x86_64-pc-windows-gnu
/c/Users/SOLO/.rustup/toolchains/stable-x86_64-pc-windows-gnu/share/doc/rustfmt
/c/Users/SOLO/.rustup/toolchains/stable-x86_64-pc-windows-gnu/share/doc/rustfmt/                         LICENSE-APACHE
/c/Users/SOLO/.rustup/toolchains/stable-x86_64-pc-windows-gnu/share/doc/rustfmt/                         LICENSE-MIT
/c/Users/SOLO/.rustup/toolchains/stable-x86_64-pc-windows-gnu/share/doc/rustfmt/                         README.md
```
So we only want an executable here, which is located in `bin`